suites/rbd/qemu/pool/ec-data-pool.yaml

```diff
@@ -0,0 +1,21 @@
+tasks:
+- exec:
+    client.0:
+      - sudo ceph osd erasure-code-profile set teuthologyprofile ruleset-failure-domain=osd m=1 k=2
+      - sudo ceph osd pool delete rbd rbd --yes-i-really-really-mean-it
+      - sudo ceph osd pool create rbd 4 4 erasure teuthologyprofile
```

Comment:
I kind of misunderstood your original comment to just have `pool` directory for `qemu` tests -- now I've just merged `qemu/cachepool` with `qemu/datapool` as `qemu/pool`, no data pool over cache pool tests...

Now that we do not want to do datapool over cache pool testing (I understand the reason now), my guess is that this needs to be done for other tests which introduce data pool (cli, librbd) or is it that qemu tests are the only exception for some reason (long running, etc..)?